Source/VolunteerReporting/Events/External/HealthRiskCreated.cs

```diff
@@ -12,5 +12,12 @@ public class HealthRiskCreated : IEvent
         public Guid Id { get; set; }
         public int ReadableId { get; set; }
         public string Name { get; set; }
+        public int? Threshold { get; set; }
+        public string ConfirmedCase { get; set; }
+        public string Note { get; set; }
+        public string SuspectedCase { get; set; }
+        public string ProbableCase { get; set; }
+        public string CommunityCase { get; set; }
```

Comment:
Do we need Threshold, ConfirmedCase, Note, SuspectedCase or ProbableCase in VolunteerReporting?

The KeyMessage we need as this should be included in the messages sent from the system to the data collectors.

Reply:
I also thought about this, but came to the (preliminary) conclusion that the event will however have all attributes from where it was raised, so event if the event processor and read model is just utilizing a very few, the event class itself should be as close to the event as possible. Please chip in and correct me, @einari

Reply:
I've created a separate issue for the discussion on whether all properties should be included on the event or not: #232.